backend/benefit/applications/api/v1/ahjo_integration_views.py

```diff
@@ -150,10 +153,14 @@ def handle_success_callback(
             elif request_type == AhjoRequestType.SEND_DECISION_PROPOSAL:
                 ahjo_status = AhjoStatusEnum.DECISION_PROPOSAL_ACCEPTED
                 info = "Decision proposal was sent to Ahjo"
-            elif request_type == AhjoRequestType.UPDATE_APPLICATION:
-                self._handle_update_records_success(application, callback_data)
+            elif request_type in AhjoRequestType.UPDATE_APPLICATION:
```

Comment:
Why is this `in` while the others are `==`?

Reply:
Ahh I think I initially compared it with a list of AhjoRequestTypes, and left it like that.